src/components/PredictionGame.tsx

```
@@ -177,306 +219,372 @@ interface AssignmentPredictionProps {
+  const hasAllGuesses =
+    hosts.length > 0 && hosts.every((host) => !!getGuessForHost(host.id));
+  const isCollapsed = !userExpanded;
+
+  return (
+    <div className="relative flex flex-col gap-4">
+      {/* Collapsed Header View */}
+      {isCollapsed ? (
+        <div
+          className="flex cursor-pointer items-center gap-2 rounded-lg border border-gray-700/50 bg-gray-800/40 p-2 transition-colors hover:bg-gray-800/60 sm:gap-4 sm:p-3"
+          onClick={() => setUserExpanded(true)}
+        >
```

Biome (2.4.7)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)